src/megaapi_impl.cpp

```diff
@@ -7444,6 +7444,17 @@ void MegaApiImpl::abortPendingActions(error preverror)
     // -- Transfers in progress --
     {
         DBTableTransactionCommitter committer(client->tctable);
+
```

Comment:
Note for reviewers: we move the cleanup of the queued transfers before the cleanup of ongoing transfers because, otherwise, finishing the the ongoing folder transfer will destroy the object, and it will delete queued transfers looking for them in the queue. However, if the queue is empty, we save some cycles.